tests/console/docker_compose.pm

```diff
@@ -42,8 +42,8 @@ sub run {
 
     # Prepare docker-compose.yml and haproxy.cfg
     assert_script_run 'mkdir -p dcproject; cd dcproject';
-    assert_script_run("wget " . data_url("containers/docker-compose.yml") . " -O docker-compose.yml");
-    assert_script_run("wget " . data_url("containers/haproxy.cfg") . " -O haproxy.cfg");
+    assert_script_run("curl -O " . data_url("containers/docker-compose.yml"));
```

Comment:
not saying that curl is worse than wget but just curious what motivation do you have here ?

Reply:
JeOS doesn't have `wget` preinstalled

Reply:
ok , good enough :)

Reply:
same as https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10379